File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.js
function solution(age) {
    var answer = '';
    // 숫자를 문자로 변환 후 하나씩 꺼내서 알파벳과 매칭해줌
    const stringAge = String(age);
    let cnt = 0;
    while(cnt < stringAge.length) {
        answer = answer + String.fromCharCode(stringAge[cnt].charCodeAt(0) + 49);
        cnt = cnt + 1;
    }
    return answer;
}

File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.java
import java.util.*;
class Solution {
    public String solution(int age) {
        // split(regex, limit)
        // substring(startIndex, endIndex)
        // substr(start, length)
        String answer = "";

        // 숫자형을 문자형으로 
        String s = String.valueOf(age);
        // 문자형을 나눈다
        String[] arr = s.split("");
        
        // 아스키코드 개념
        for(int i = 0; i < arr.length; i++) {
            answer += ((char)((Integer.parseInt(arr[i]) + 97)));
        }
        return answer;
    }
}